SU2_CFD/include/output/CHeatOutput.hpp

```diff
@@ -78,11 +84,18 @@ class CHeatOutput final: public COutput {
    * \param[in] solver - The container holding all solution data.
    * \param[in] iPoint - Index of the point.
    */
-  void LoadVolumeData(CConfig *config, CGeometry *geometry, CSolver **solver, unsigned long iPoint) override; 
-  
+  void LoadVolumeData(CConfig *config, CGeometry *geometry, CSolver **solver, unsigned long iPoint) override;   
+
   /*!
-   * \brief Set the available history output fields
-   * \param[in] config - Definition of the particular problem.
+   * \brief LoadSurfaceData
+   * \param config
```

Comment:
thanks for keeping the parameters documented, I know in this case it is obvious but the param type [in] is missing

Reply:
Added.